tests/vmi_multus_test.go

```diff
@@ -853,16 +856,44 @@ var _ = Describe("[Serial]SRIOV", func() {
 		It("[test_id:3956]should connect to another machine with sriov interface over IPv4", func() {
 			cidrA := "192.168.1.1/24"
 			cidrB := "192.168.1.2/24"
+			cidrDebug := "192.168.0.100/24"
+			const maxCommandTimeout = 10 * time.Second
+
+			//add ip for the pf to debug ping
+			_, err := RunOnNode("sriov-worker", fmt.Sprintf("ip addr add %s dev enp4s0f0", cidrToIP(cidrDebug)))
+			Expect(err).ToNot(HaveOccurred())
+
 			//create two vms on the smae sriov network
 			vmi1, vmi2 := createSriovVMs(sriovnet3, sriovnet3, cidrA, cidrB)
 
-			assert.XFail("suspected cloud-init issue: https://github.com/kubevirt/kubevirt/issues/4642")
-			Eventually(func() error {
-				return libnet.PingFromVMConsole(vmi1, cidrToIP(cidrB))
-			}, 15*time.Second, time.Second).Should(Succeed())
+			//assert.XFail("suspected cloud-init issue: https://github.com/kubevirt/kubevirt/issues/4642")
 			Eventually(func() error {
-				return libnet.PingFromVMConsole(vmi2, cidrToIP(cidrA))
-			}, 15*time.Second, time.Second).Should(Succeed())
+				err1 := multierr.Combine(libnet.PingFromVMConsole(vmi1, cidrToIP(cidrB)), libnet.PingFromVMConsole(vmi2, cidrToIP(cidrA)),
+					libnet.PingFromVMConsole(vmi1, cidrToIP(cidrDebug)), libnet.PingFromVMConsole(vmi2, cidrToIP(cidrDebug)))
+
+				_, err2 := RunOnNode("sriov-worker", fmt.Sprintf("ip addr del %s dev enp4s0f0", "192.168.0.100"))
+				err = multierr.Append(err1, err2)
+
+				vm1respIPRoute, _ := console.RunCommandWithResponse(vmi1, "ip route", maxCommandTimeout)
+				vm1respIPLink, _ := console.RunCommandWithResponse(vmi1, "ip link", maxCommandTimeout)
+				vm1respIPAddress, _ := console.RunCommandWithResponse(vmi1, "ip address", maxCommandTimeout)
+				vm1respIPNeigh, _ := console.RunCommandWithResponse(vmi1, "ip neigh", maxCommandTimeout)
```

Comment:
Could you please run those before the ping command
we better check what happen before the ping as well

Reply:
Done